cypress/integration/discountTest.js

```diff
@@ -0,0 +1,31 @@
+import HomePage from '../../src/pages/HomePage'
+import DiscountPage from '../../src/pages/DiscountPage'
+import * as urls from '../../testData/urls'
+
+describe("Verification of the discount sum", ()=>{
+    let priceOneValue = 0
+    let priceTwoValue = 0
+    let priceResultValue = 0
+    let testResultValue = 0
```

Comment:
No need to create extra variable

Reply:
should I move them in test? I deleted testResultValue variable

Reply:
Sorry, I mean line 9 only.
Delete variable in line 9.
Also re lines 6-8 - if you use variables in a single test - move them in test.
If you will use them in several tests - leave as it is - to avoid code duplication.

Reply:
I moved them to getPrices function. Is it ok?

Reply:
If it works for our purpose - then yes.